kotlin/README.md

```diff
@@ -80,4 +80,4 @@ can not use things like `forEach` because of ambiguity between `Iterable<?>.forE
 (`foreach` from Spark's API does work, though).
 * The scoped mode for Kotlin Spark Interpreter currently has issues with having the same class output directory 
 for different intepreters, leading to overwriting classes. Adding prefixes to generated classes or putting them
- in separate directories leads to `ClassNotFoundException` on Spark executors.
\ No newline at end of file
+ in separate directories leads to `ClassNotFoundException` on Spark executors.
```

Comment:
What changes did you make on this line ?

Reply:
Nothing, i think the Github editor added a new line at the end of file